examples/packages/express-graphql/package.json

```diff
@@ -18,9 +18,17 @@
     "access": "restricted"
   },
   "dependencies": {
-    "express-graphql": "0.12.0"
+    "@graphql-authz/core": "^1.3.1",
```

Comment:
please, use exact version to be consistent. also, please check other places as well